src/edit/tools/L.tools.EXIF.js

```diff
@@ -0,0 +1,62 @@
+L.tools = L.tools || {};
+
+L.tools.EXIF = function getEXIFdata(ref, overlay) {
+  var GPS = ref.exifdata,
+    lat,
+    lng;
+  
+  if (
+    typeof GPS.GPSLatitude !== "undefined" &&
+    typeof GPS.GPSLongitude !== "undefined"
+  ) {
+    lat =
+      GPS.GPSLatitude[0] + GPS.GPSLatitude[1] / 60 + GPS.GPSLatitude[2] / 3600;
+    lng =
+      GPS.GPSLongitude[0] +
+      GPS.GPSLongitude[1] / 60 +
+      GPS.GPSLongitude[2] / 3600;
+
+    if (GPS.GPSLatitudeRef !== "N") {
+      lat = lat * -1;
+    }
+    if (GPS.GPSLongitudeRef === "W") {
+      lng = lng * -1;
+    }
+  }
+
+  var angle = 0;
+
+  if (GPS.GPSImgDirectionRef === "T" || GPS.GPSImgDirectionRef === "M") {
+    angle =
+      (Math.PI / 180) *
+      (GPS.GPSImgDirection.numerator / GPS.GPSImgDirection.denominator - 90);
+  }
+
+  if (GPS.Orientation === 6) {
+    angle += (Math.PI / 180) * -90;
+  } else if (GPS.Orientation === 8) {
+    angle += (Math.PI / 180) * 90;
+  } else if (GPS.Orientation === 3) {
+    angle += (Math.PI / 180) * 180;
+  }
+
+  var panTo = L.latLng(lat, lng);
+
+  var x_diff = overlay.getCorner(0).lng - overlay.getCorner(1).lng; // width
```

Comment:
Lets update this and below to deltaX and deltaY to abide to JS camelcase convention?